CONTRIBUTING.md

```diff
@@ -1,54 +1,124 @@
+make test/e2e E2E_TEST_ARGS='--ginkgo.skip="Simple\ OVS*"'
+```
+
+## Publishing containers
+
+```shell
+# Push nmstate-handler container to remote registry
+make push-handler
+```
+
+It is possible to adjust the built container images with the following
+environment variables.
+
+```shell
+IMAGE_REGISTRY # quay.io
+IMAGE_REPO # nmstate
+
+HANDLER_IMAGE_NAME # kubernetes-nmstate-handler
+HANDLER_IMAGE_TAG # latest
+```
+
+## Manifests
+
+The operator `operator.yaml` manifest from the `deploy` folder is a template to
+be able to replace the with correct docker image to use.
+
+Everytime cluster-sync is called it will regenerate the operator yaml with
+correct kubernets-nmstate-handler image and apply it.
 
-[Development guide](docs/developer-guide-commands.md) is a go-to reference point for
-development helper commands, building, testing, container images and local
-cluster.
 
 # Open a Pull Request
-Kubernetes-nmstate generally follows the [standard github pull request process](https://gist.github.com/Chaser324/ce0505fbed06b947d962), but there is a layer of additional specific differences:
 
-The first difference you'll see is that a bot will begin applying structured labels to your PR.
+kubernetes-nmstate generally follows the [standard github pull request
+process](https://gist.github.com/Chaser324/ce0505fbed06b947d962), but there is a
+layer of additional specific differences:
 
-The bot may also make some helpful suggestions for commands to run in your PR to facilitate review.
-These /command options can be entered in comments to trigger auto-labeling and notifications. Refer to its command reference documentation.
+The first difference you'll see is that a bot will begin applying structured
+labels to your PR.
+
+The bot may also make some helpful suggestions for commands to run in your PR to
+facilitate review. These `/command` options can be entered in comments to
+trigger auto-labeling and notifications. Refer to its command reference
+documentation.
 
 Common new contributor PR issues are:
 
-Missing DCO sign-off:
-    Developers Certificate of Origin (DCO) Sign-off is a requirement for getting patches into the project (see [Developers Certificate of Origin](https://developercertificate.org/)).
-    You can "sign" this certificate by including a line in the git commit of "Signed-off-by: Legal Name <email-address>".
-    If you forgot to add the sign-off, you can also amend your commit with the sign-off:
-    `git commit --amend -s`.
+- Missing DCO sign-off:\
+  Developers Certificate of Origin (DCO) Sign-off is a requirement for getting
+  patches into the project (see [Developers Certificate of
+  Origin](https://developercertificate.org/)). You can "sign" this certificate
+  by including a line in the git commit of "Signed-off-by: Legal Name
+  <email-address>". If you forgot to add the sign-off, you can also amend your
+  commit with the sign-off: `git commit --amend -s`.
```

Comment:
The git config command to have this automatic would be nice too.

Reply:
Ditto